README.md

```diff
@@ -38,6 +38,50 @@ export default ListCardComponent.extend({
 });
 ```
 
+This components supports filtering and sorting 
+```js
+// components/customers/list-card.js
+import Ember from 'ember';
+import ListCardComponent, { QueryToken, QueryFilter, FilterGroup } from 'ember-list-card/components/list-card';
```

Comment:
not really, the `ListCardComponent` is the default so it can be imported either way